src/grpc/infer_handler.h

```diff
@@ -1076,6 +1081,21 @@ class InferHandlerState {
     if (pstr != nullptr) {
       delay_process_ms_ = atoi(pstr);
     }
+    const char* nstr = getenv("TRITONSERVER_DELAY_GRPC_NOTIFICATION");
```

Comment:
Will refactor variable names in another gRPC improvement PR.

Reply:
would this be better to re-name them now? then there's no need to go back with refactor

Reply:
I kept the naming style to avoid changes to lines above which look distracting. There will be another PR incoming which I will add more loggings to the gRPC handler that allows easier debugging, as well as variables renaming.

Reply:
This part is refactored.